### PR TITLE
updated card styles

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Dual Merge: [BE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/665)

Before:
image

After:
image

### CODE REVIEW

**Comment by SidelineCory24** [comment]
Is this the intended design for this positions `Information...`?
image

**Comment by elizabeth-jimenez** [comment]
> Is this the intended design for this positions `Information...`? image

Yes and no.
Those titles are capped at 15 chars, but they dont take into account the 30 char limit

File: src/Components/Agenda/AgendaItemCard/AgendaItemCard.jsx [hunk 1]
```diff
@@ -28,15 +26,32 @@ const AgendaItemCard = props => {
...
 
-  const formatStr = (a) => shortenString(a, 15);
+  const renderTitles = () => {
+    let capAt = 13;
+    const title1 = get(legs$, '[0].pos_title') || 'N/A';
+    const title2 = get(legs$, '[0].pos_title') || 'N/A';
+    const t1L = title1.length;
+    const t2L = title2.length;
+    // does either have chars to give up and does either want them
+    if ((t1L < 13 || t2L < 13) && (t1L > 13 || t2L > 13)) {
```
Comment (SidelineCory24) [inline src/Components/Agenda/AgendaItemCard/AgendaItemCard.jsx:36]: 🙌

**Comment by SidelineCory24** [comment]
@elizabeth-jimenez there's a code climate issue but I don't see anything in the details. Maybe one of the test files needs to be updated?

**Comment by elizabeth-jimenez** [comment]
> @elizabeth-jimenez there's a code climate issue but I don't see anything in the details. Maybe one of the test files needs to be updated?

It's the coverage - i'll look into it

**Review by SidelineCory24 — APPROVED** [review]
Looks good to go! 🚀

**Pull request merged**